docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Exit on error
+set -e
+
+# use DEBUG=echo ./docker.sh to print all commands
+export DEBUG=${DEBUG:-""}
+export MAPBOX_TOKEN=${MAPBOX_TOKEN:-""}
+
+# Build docker image
+$DEBUG docker build --build-arg MAPBOX_TOKEN="${MAPBOX_TOKEN}" -t oceans-1876/frontend .
```

Comment:
What does `DEBUG` do in this context? Is it a flag for the `docker` command?

Reply:
You can set "echo" in the environment to print the command. Could be useful for debugging issues with passing in the build args if there are issues with getting the environment injected to the container being built.

Reply:
Interesting. Good to know. And I just noticed the comment in the script about `DEBUG`!